frontend/src/contracts-typechain/factories/Job__factory.ts

```diff
@@ -11,7 +11,17 @@ const _abi = [
     inputs: [
```

Comment:
Let's ignore these files please ! I want our PRs to be clean.

Reply:
Created #89